ipc/apis.go

```diff
@@ -103,15 +102,14 @@ func ContainerRegister(ctx interface{}, data *grpc.ContainerData) error {
 		return err
 	}
 
-	logrus.Infof("Container registration successfully completed for id: %s",
-		data.Id)
+	logrus.Infof("Container registration: %v", cntr)
```

Comment:
Please use "Container registration completed" description instead to indicate that process completed successfully (otherwise it could be also understood as the container-registration msg being received).